package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scripts": {
+    "prepare": "husky install && chmod ug+x .husky/*"
+  },
```

Comment:
You won't need to add this. You can change the mode on your side and commit, that will be fine ig!

Reply:
I don't see any changes in my project after changing the mode locally, I think it might be better to keep this line.

Reference: [https://stackoverflow.com/questions/68367259/husky-needs-to-make-executable-for-every-new-branch](url)

Let me know if I'm getting something wrong here

Reply:
> I don't see any changes in my project after changing the mode locally, I think it might be better to keep this line.
> 
> Reference: [https://stackoverflow.com/questions/68367259/husky-needs-to-make-executable-for-every-new-branch](url)
> 
> Let me know if I'm getting something wrong here

It's happening, let me see if I can push the changes

Reply:
I have updated the mode, It's now working fine !

Reply:
Okay, I'll keep that in mind